llvm/include/llvm/CodeGen/AccelTable.h

```diff
@@ -312,15 +355,20 @@ void emitDWARF5AccelTable(AsmPrinter *Asm, DWARF5AccelTable &Contents,
                           const DwarfDebug &DD,
                           ArrayRef<std::unique_ptr<DwarfCompileUnit>> CUs);
 
+struct DWARF5AccelTableEntry {
```

Comment:
DWARFAccelTableData V DWARF5AccelTableEntry is a bit subtle/hard to know what's the difference

Could be called something like `DWARF5AccelTableUnitIndexAndEncoding`? (maybe a member of DWARF5AccelTable and then it could be DWARF5AccelTable::UnitIndexAndEncoding (or UnitIndexAndForm))